src/qtt/measurements/scans.py

```diff
@@ -2310,23 +2311,21 @@ def plotData(alldata, diff_dir=None, fig=1):
         imx = qtt.utilities.tools.diffImageSmooth(alldata.measured.ndarray, dy=diff_dir)
         name = 'diff_dir_%s' % diff_dir
         name = uniqueArrayName(alldata, name)
-        data_arr = qcodes.DataArray(name=name, label=name, array_id=name,
+        data_arr = DataArray(name=name, label=name, array_id=name,
```

Comment:
You didn't correct the next line(s) for the changed indentation.

Reply:
Did an autopep on the file.